api/providers/tasks.py

```diff
@@ -336,44 +287,123 @@ def handle_registration_row(row, initiator, provider, schema, auto_approval=Fals
     auth = Auth(user=initiator)
 
     # Check node
-    node = None
     node_id = metadata.get('Project GUID')
+    node = check_node(node_id, initiator, row, row_title, row_external_id)
```

Comment:
All helper methods need three arguments `row`, `row_title`, `row_external_id` in order to set the `RegistrationBulkCreationRowError` exception.